docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -65,27 +65,40 @@ The following fields are optional for `require_session_join`:
 |---|---|---|
 |`on_leave`|[On leave](#on-leave)|The action to take when the policy is no longer satisfied|
 
-#### Example
+#### Example Session Moderation Required Role
 
 The policy below specifies that the `prod-access` role must have a minimum of
 two users with the role `auditor` and the mode `moderator` present in the
 session to start it. The policy applies to SSH and Kubernetes sessions only.
 
 When a user with this require policy starts a session, it will be pending until
-the policy is fulfilled. 
+the policy is fulfilled. The role matches to `env: prod` labeled `ssh` and `k8s`
+resources. 
 
```

Comment:
Let's add a plain-text description of what this policy is doing. Something like:

> This policy requires that at least one user with the `auditor` role is present in order for SSH or Kubernetes sessions to begin.